mkimage.sh

```diff
@@ -100,6 +100,12 @@ ENV DEBIAN_FRONTEND noninteractive
 LABEL io.resin.architecture="rpi" \
 	  io.resin.qemu.version="$RESIN_QEMU_VERSION" \\
 	  io.resin.device-type="raspberrypi"
+RUN apt-get update && apt-get install -y --no-install-recommends wget \
+	&& wget http://archive.raspberrypi.org/debian/raspberrypi.gpg.key \
+	&& apt-key add raspberrypi.gpg.key \
+	&& rm -rf raspberrypi.gpg.key \
+	&& echo "deb http://archive.raspberrypi.org/debian/ jessie main" >> /etc/apt/sources.list \
```

Comment:
@petrosagg It comes from this thread https://gitter.im/resin-io/chat?at=57323356a351d8310952c20b. An user wanted to install some rpi specific packages but they are not available from this mirror `http://archive.raspbian.org/raspbian jessie main contrib non-free rpi firmware`. So I think we should add it to our raspberrypi base images